packages/docusaurus-theme-classic/src/types.d.ts

```diff
@@ -73,11 +73,11 @@ declare module '@theme/DocPaginator' {
 }
 
 declare module '@theme/DocSidebar' {
-  import type {PropSidebarItem} from '@docusaurus/plugin-content-docs-types';
+  // import type {PropSidebarItem} from '@docusaurus/plugin-content-docs-types';
```

Comment:
This is the issue I am having with deployment, since I'm adding a new field to sidebar.js its falling the test is there somewhere where I can add my new `link` field?

Reply:
Not sure what you are talking about

The CI deployment error is currently:

![image](https://user-images.githubusercontent.com/749374/102238925-99e91b80-3ef6-11eb-9511-c55b7d97ff68.png)


This is an error emitted on purpose by the docs plugin, where you should allow/validate the new `link` attribute